worker/draft.go

```diff
@@ -832,6 +832,7 @@ func (n *node) commitOrAbort(pkey uint64, delta *pb.OracleDelta) error {
 		if err != nil {
 			glog.Errorf("Error while applying txn status to disk (%d -> %d): %v",
 				start, commit, err)
+			panic(err)
```

Comment:
can we add more logs here, enough to figure out the issue?

Reply:
I think panic is the most graceful way to die! The other things that we could do is retry but we already doing that.